cultcargo/builder/cargo-manifest.yml

```diff
@@ -28,6 +28,13 @@ images:
     versions:
       base: {}
 
+  aimfast:
+    versions:
+      '1.3.4':
+        package: aimfast==1.3.4
+      base:
```

Comment:
Why not call it 'master' or 'main', for consistency with the other packages? (Meaning tracking the master or main branch).

Reply:
duly